news/_posts/2020-06-12-product-features-methodology-at-prestashop.md

```diff
@@ -0,0 +1,109 @@
+Right now, collecting feedback is not difficult, Github is probably the main source, but there are also events such as [ambassadors' meetups](http://ambassadors.prestashop.com/ambassadors) and [PSD](https://events.prestashop.com/), user interviews regularly conducted by the product team, and the [community Slack](https://github.com/PrestaShop/open-source/blob/master/slack/readme.md).
+The team also keeps an eye on what’s happening around in the e-commerce and CMS industries to make sure the software goes in the right direction regarding the market's evolutions and expectations.
+
+Collecting feedback is easy. What is more difficult is to centralize and analyze it in order to identify the users’ needs lying behind.
+
+Productboard will help the team here. By making it easier and automatic to centralize feedback directly in the same place. Wherever it comes from.
+The team calls these feedback **insights**. On Productboard, product managers can create topics to group **insights** together, this step helps form a first **expression of needs**. Sometimes the expression of needs already gives the best solution, sometimes it is still a bit early to know how it could be solved.
+
+What’s in it for the community? Well Productboard also makes it possible to share that externally.
+This is great to inform the community about what the team has identified, and also to keep collecting more information about it to improve the team’s understanding on this problem.
+**You are now able to see these expressions of needs on the PrestaShop Feedback website.**
+
+### How to go from a need to a feature?
+
+Centralizing and analyzing insights is difficult.
+Well, the next step in product development is even more difficult: prioritizing the users' needs.
+
+PrestaShop is used on 300k+ online stores around the world, and its community gathers more than 1 million people.
+There are hundreds of needs out there, waiting to be solved. So how does the team decide what to build next?
+
+The Core product team uses several criteria to evaluate the importance of a need. There is no scientific, or magical, formula. It is a list of aspects that the team reviews in light of the research and analysis previously run. Among these aspects are:
+
+- legal requirements,
+- the impact on the merchants’ businesses,
+- the impact on merchant productivity,
+- the impact on the developer experience,
+- and many more...
+
+As you can see, it is important for the team to take into account all the impacts of solving a need, and building features.
+And that is something that Productboard makes easier too.
+
+In the PrestaShop Feedback space, as a visitor, it is possible not only to add comments but also to rate the impact of an expression of needs or a feature on oneself. Three values exist: Nice-to-have, Important, and Critical.
+This rating is directly taken into account in Productboard and gives a **user score**. More about it in the second part.
```

Comment:
@colinegin, should we keep it as well?

Reply:
I guess we talk about it in the next article ? 
Maybe what we can say is just that we user customers feedbacks at all stages of research and conception without saying much about PrestaShop Feedback ?

Reply:
cf. line 77. If it suits you fine, I'll delete this paragraph and keep it for the Productboard post. :-)